.evergreen/run-oidc-prose-tests.sh

```diff
@@ -19,10 +19,16 @@ if [ "$ENVIRONMENT" = "azure" ]; then
   npm run check:oidc-azure
 elif [ "$ENVIRONMENT" = "gcp" ]; then
   npm run check:oidc-gcp
-else
+elif [ "$ENVIRONMENT" = "test" ]; then
   if [ -z "${OIDC_TOKEN_FILE}" ]; then
     echo "Must specify OIDC_TOKEN_FILE"
     exit 1
   fi
   npm run check:oidc-test
+else
+  if [ -z "${K8S_VARIANT}" ]; then
```

Comment:
What is this variable used for?

Reply:
This variable is set in CI to either "AKS", "GKE", or "EKS" and depending on it's value the scripts in https://github.com/mongodb-labs/drivers-evergreen-tools/tree/master/.evergreen/auth_oidc/k8s will set up the appropriate k8s pod in that environment that it will self test and then run our tests in.